modules/azure/k8s/outputs.tf

```diff
@@ -32,9 +32,13 @@ resource "local_file" "kubeconfig" {
 }
 
 output "cluster_name" {
-  value = var.cluster_name
+  value = azurerm_kubernetes_cluster.k8s.name
+}
+
+output "kubelet_identity" {
+  value = azurerm_kubernetes_cluster.k8s.kubelet_identity
 }
 
 output "locality_region" {
   value = var.location
-}
+}
```

Comment:
I see no script to cleanup records created by external-dns in Azure. Are they cleaned up?

Reply:
yes, azure apis do allow zone termination with the existing records.